### PR TITLE
Btc destination address base58

### PR DESCRIPTION
**Pull request by kelvinator07**

### CODE REVIEW

**Review by jeremy-then — APPROVED** [review]
LGTM.

File: sample/sample.js [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+    bitcoin.payments.p2pkh({
+        hash: Buffer.from(stripHexPrefix(pubKeyHash), 'hex'),
+        network: getNetwork(network)
+    }).address;
+
+/* eslint no-param-reassign: "off" */
+const formatBtcDestinationAddressToBase58 = (pegoutRequests, network) =>
+    pegoutRequests.map(pegout => {
+        pegout.destinationAddressBase58 = btcAddressFromPublicKeyHash(pegout.destinationAddressHash160, network);
+        delete pegout.destinationAddressHash160;
```
Comment (marcos-iov) [inline sample/sample.js:20]: Perhaps we could leave this field, just in case

File: sample/sample.js [hunk 2]
```diff
@@ -13,7 +32,7 @@ const { getBridgeState } = require('../index');
         console.log(`Active federation UTXOs (${bridgeStateResult.activeFederationUtxos.length})`);
         console.log(bridgeStateResult.activeFederationUtxos);
         console.log(`Peg-out requests (${bridgeStateResult.pegoutRequests.length})`);
-        console.log(bridgeStateResult.pegoutRequests);
+        console.log(formatBtcDestinationAddressToBase58(bridgeStateResult.pegoutRequests, process.argv[2]));
```
Comment (marcos-iov) [inline sample/sample.js:35]: What happens when we run the sample against localhost or an alphanet node ip?

Maybe use testnet by default?

Comment (kelvinator07) [reply sample/sample.js:35]: Yes Testnet Is The Default

File: sample/sample.js (hunk 1 above)
Comment (kelvinator07) [reply sample/sample.js]: Okay Updated

**Review by marcos-iov — APPROVED** [review]

File: sample/sample.js (hunk 2 above)
Comment (marcos-iov) [reply sample/sample.js:34]: I see, thanks

**Pull request merged**